sketch.js

```diff
@@ -116,13 +116,14 @@ function sketch_header_background(p) {
       //this.qt.draw();
 
       // the connects
+      p.stroke(255, 255, 255, 1);
       for (const pair of this.connects){
         let d = p.map(distance(pair[0].x, pair[0].y, pair[1].x, pair[1].y), 0, 100, 0.8, 0);
-        p.stroke(255, 255, 255, d);
-        p.strokeWeight(d);
+        //p.stroke(255, 255, 255, d);
+        p.strokeWeight(d * d);
```

Comment:
If `d` is 0.5, this will essentially cut the weight in half twice, instead of cutting the weight in half and the opacity in half independently. This achieves a result that is visually almost identical, since the lines are already so close to 1 pixel in width.